core/unit_test/TestViewAPI.hpp

```diff
@@ -1002,19 +1002,20 @@ class TestViewAPI {
     hView3 hv_3("dView3::HostMirror", N0);
     hView4 hv_4("dView4::HostMirror", N0);
 
-    dView0 dv_0_1(nullptr);
+    dView0 dummy("dummy");
+    dView0 dv_0_1(dummy.data());
```

Comment:
I get that the rank-0 view case is not allowed but not the subsequent changes.

Reply:
N0 is not zero so those are not empty Views either.